app/models/case_log.rb

```diff
@@ -49,6 +49,30 @@ def validate_armed_forces_injured(record)
     end
   end
 
+  def validate_household_pregnancy(record)
+    if (record.pregnancy == "Yes" || record.pregnancy == "Prefer not to say") && !women_of_child_bearing_age_in_household(record)
+      record.errors.add :pregnancy, "You must answer no as there are no female tenants aged 16-50 in the property"
+    end
+  end
+
+  def women_of_child_bearing_age_in_household(record)
+    unless record.tenant_gender.nil? || record.tenant_age.nil?
+      if record.tenant_gender == "Female" && record.tenant_age >= 16 && record.tenant_age <= 50
```

Comment:
This can just be:

```suggestion
      record.tenant_gender == "Female" && record.tenant_age >= 16 && record.tenant_age <= 50
```

Don't need the `if` or the `return`